### PR TITLE
Add Contribution guide and Issue/PR templates

### PR DESCRIPTION
**Pull request by paulbalaji**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](https://github.com/spatialos/gdk-for-unity#give-us-feedback).
```
Comment (firtina-improbable) [inline .github/pull_request_template.md]: CONTRIBUTING.md missing

Comment (paulbalaji) [reply .github/pull_request_template.md]: it's include in the PR? 

.github/CONTRIBUTIONS.md

Comment (paulbalaji) [reply .github/pull_request_template.md]: ah yes the link will be broken on the PR til it's merged

Comment (firtina-improbable) [reply .github/pull_request_template.md]: Ah I see!

**Comment by paulbalaji** [comment]
Part 3 of 3
https://github.com/spatialos/gdk-for-unity/pull/571
https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/54
https://github.com/spatialos/gdk-for-unity-blank-project/pull/3

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Repository contributions
+
+## Contributions policy
+We are currently not accepting public contributions (but we do plan to in later releases and we’ll keep you osted on this). However, we are taking issues and we do want your [feedback](../README.md#give-us-feedback).
+
+## Coding standards
+See the [GDK for Unity C# coding standards guide](../coding-standards.md).
```
Comment (jessicafalk) [inline .github/CONTRIBUTING.md]: link is broken

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+## Getting started
+See [About the SpatialOS Unity GDK](https://docs.improbable.io/unity/alpha/get-started) for details on how to get started using the Unity GDK.
```
Comment (jessicafalk) [inline .github/CONTRIBUTING.md]: Unity GDK 😱

**Pull request merged**